### ISSUE TITLE
Conformance to EAA

### ISSUE THREAD

**Issue by LaurenceZaysser** [w3c/a11y-discov-vocab#197]
Though the EAA (European Accessibility Act) will come into force next June 28th, I still do not see any mention of the conformance to this standard.  European customers will shortly ask for this label (in remplacement with the LIA one probably).

https://w3c.github.io/a11y-discov-vocab/crosswalk/#conformance-and-exemption-declarations
https://ns.editeur.org/onix/en/196

**Comment by chrisONIX** [comment]
The only standard associated with EU Accessibility  is EN 301 549 - which incorporates WCAG 2.1 Level AA criteria. This is is the standard associated with the EU Web Accessibility Directive ( more about EN 301 549 here  https://en.wikipedia.org/wiki/EN_301_549 

It is already possible to say that a publication conforms to WCAG and what level. The EU has no set standard(s) of conformance associated with the EAA in regards to digital publications beyond what was already associated with the older Web Accessibility Directive as far as I am aware?   

The publisher or creator of an EPUB or other digital product can use existing metadata to specify conformance and certification to recognised standards like WCAG or to point to other certification and compliance schemes.

Does anyone know if there is an official body in any of the EU countries that is certifying products yet and issuing a “label” that a product is in conformance with both European Union legislation and national legislation in regards to the EAA and other that could be referred to? 

A claim that a product conforms to particular legislation also carries legal implications in regards to consumer rights and liabilities for producers in what they are claiming, as well as for any public site displaying this information, so any addition should be something that references an official body and a transparent process for certification. 

NB the existing LIA code in ONIX will not be replaced as this would break backwards compatibility of the ONIX standard, any reference to any official body certifying products can use existing ONIX codes 90 and 93. 
If there are any new bodies or EU wide certification processes they could also get a new ONIX code if necessary.

**Comment by LaurenceZaysser** [comment]
The compliance EAA stamp would be equivalent of the CE marking/stamp on physical products.

The spirit of the EAA has been to define functional high level criteria in order to ensure the stability of European accessibility requirements on products and services. In fact, standards such as WCAG constantly change. So will the references to WCAG versions (move to WCAG 2.3, 2.4, ... , 3.0, etc.). WCAG version does not speak to most customers, as it is too technical.
Moreover, WCAG is restricted to W3C standards, when other digital formats exist/will exist. In this case
the conformance to the EAA cannot be inferred from any WCAG label.

Yes indeed, an EAA label will have legal impacts . As far as I know from my experience in Editis Publishing group, 
the organizations legally responsable for accessibility declaration are the publishers and their distributors. Not the resellers (except when adding metadata of there own!). That is why, on my opinion, resellers should better not take the risk of calculating an EAA label...

**Comment by LaurenceZaysser** [comment]
https://www.w3.org/TR/epub-a11y-eaa-mapping

**Comment by LaurenceZaysser** [comment]
Please note that there no perfect bijection between WCAG AA and the EAA.
Most of the time the WCAG AA is over the EAA. But sometimes, it is underneath.
[https://github.com/w3c/wcag/issues/4477](https://github.com/w3c/wcag/issues/4477)
